ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -705,6 +706,11 @@ export default defineComponent({
       { deep: true }
     )
 
+    // Ensures filtering is cleared when returing to dashboard from registrations
+    onBeforeMount(() => {
+      clearFilters()
+    })
+
```

Comment:
Is this a request by design?

Reply:
No I can remove this, but I thought it was strange behaviour that when you return from a registration at the moment, it leaves the filter columns showing even though it is not filtering.

![image](https://github.com/bcgov/ppr/assets/77707952/c320aaaf-6171-45fb-a3d8-ddd9c147d23f)

Reply:
Ah i see. 
Yea, i think that makes sense to clear, the above scenario is definitely confusing.